src/openzaak/management/commands/migrate_vestigingen_to_nnps.py

```diff
@@ -0,0 +1,57 @@
+# SPDX-License-Identifier: EUPL-1.2
+# Copyright (C) 2025 Dimpact
+from django.core.management import BaseCommand
+from django.db import transaction
+
+from openzaak.components.zaken.models import NietNatuurlijkPersoon, Vestiging
+
+
+class Command(BaseCommand):
+    help = "Migrate deprecated Vestiging rollen to NietNatuurlijkePersoon."
+
+    @transaction.atomic
+    def handle(self, *args, **options):
+        self.stdout.write(
+            f"Attempting to migrate {Vestiging.objects.count()} Vestigingen..."
+        )
+
+        migration_count = 0
+        for vestiging in Vestiging.objects.all():
```

Comment:
Generally when iterating over a queryset, it's better to use `.iterator()` over `.all()`, because when using `.all()` the entire queryset is loaded into memory. If the queryset isn't too large this won't be an issue, but if the queryset is large you can run into out of memory errors (similar to #1940)

See also: https://docs.djangoproject.com/en/5.1/ref/models/querysets/#iterator
```suggestion
        for vestiging in Vestiging.objects.iterator():
```